scripts/README.md

```diff
@@ -0,0 +1,22 @@
+# Firezone shell scripts
+
+This directory contains various shell scripts used for development, testing, and
+deployment of the Firezone product.
+
+## Developer Setup
+
```

Comment:
```suggestion
sudo apt-get install go shellcheck
go install mvdan.cc/sh/v3/cmd/shfmt@latest
```

Just put the exact commands up front. Devs will read the rest if they need. Do I really need to install Go just for this linter? :(

And I have no clue how editor plugins work, I don't use any, I'm not sure if Kate has plugins for this, so I'd offer the Bash equivalent:

```bash
shellcheck scripts/**/*.sh
shfmt -i 4 scripts/**/*.sh
```

Reply:
@ReactorScram, if it's written in `go` then it's very unlikely that you'll need to install `go` to run it.  That's one thing that I really like about `go` is that it's so easy to build static binaries for a different architectures.  I'm not sure why

Looking at the releases page, it looks like they've already pre-built the binary for a lot of different systems:
https://github.com/mvdan/sh/releases

Also, it looks like they have a decent number of package manager integrations, so that could be another option.  Not sure how up to date the package managers are though.

Reply:
Yeah turned out it's in apt, but WSL is only giving me 3.4.3. I'll change my suggestion to point at the releases page

Reply:
Thanks for the suggestions, updated